app/views/provider/admin/account/users/_form.html.erb

```diff
@@ -57,11 +57,10 @@
         <script>
           document.addEventListener('DOMContentLoaded', function () {
```

Comment:
we could use ruby `content_for` here and pass them as `data` attr, then initialize the `permissionsWidget` inside the pack.